sakura_core/cmd/CViewCommander_Grep.cpp

```diff
@@ -250,7 +250,7 @@ void CViewCommander::Command_GREP_REPLACE( void )
 		if( cDlgGrepRep.m_bGrepSeparateFolder		)wcscat( pOpt, L"D" );
 		if( cDlgGrepRep.m_bPaste					)wcscat( pOpt, L"C" );	// クリップボードから貼り付け
 		if( cDlgGrepRep.m_bBackup					)wcscat( pOpt, L"O" );	// バックアップ作成
-		if( 0 < wcslen( pOpt ) ){
+		if( pOpt[0] != '\0' ) {
```

Comment:
```suggestion
		if( pOpt[0] ) {
```

分岐条件に「否定表現」を含めると「否定表現」を含まない場合に比べて「難しい」と思う人が増えるそうです。
「`==`とか`!=`がないと読めない」って人もいるので「どちらが良いか？」の答えはありませんが。

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17